packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -64,23 +83,32 @@ function NavigationMenuSelector(
 	const hasManagePermissions =
 		canUserCreateNavigationMenu || canUserUpdateNavigationMenu;
 
+	useEffect( () => {
+		if ( ! hasNavigationMenus ) {
+			setSelectorLabel( __( 'No menus. Create one?' ) );
```

Comment:
This looks a bit unusual in practice:
<img width="270" alt="Screen Shot 2022-08-29 at 3 56 02 pm" src="https://user-images.githubusercontent.com/677833/187133513-4b74649f-73a9-419e-a243-3aa10c04424a.png">

Perhaps there needs to be some thought about what happens if a menu is missing (or deleted) in the context of this change. IMO, it should probably just fallback to whatever the default menu is (e.g. Page List, or another existing menu).

Reply:
The only problem is what happens when we display the page list fallback. If the navigation block falls back to some menu this UI change is fine. The problem is if the block displays a page list, what then? I chose this direct way to telling the user that even if they have inserted a navigation block, which displays something, they  have not "created" a menu yet.

Reply:
Yes, I'll post a recording of how it currently works and loop in design feedback as I am really stretching my opinions out on this one :D